Automattic/wp-calypso
# Email stats: add links in the click stats

#### Proposed Changes

* This PR adds a box with an overview of the clicked links to the click stats:

<img width="578" alt="CleanShot 2023-01-24 at 16 58 45@2x" src="https://user-images.githubusercontent.com/528287/214343447-86ddcb22-c574-4213-ac57-eebca336d801.png">


#### Testing Instructions

- Apply this PR
- Visit `http://calypso.localhost:3000/stats/email/clicks/<siteid>/day/<postid>?flags=newsletter/stats`

You should see the box with the links in the bottom.

#### Pre-merge Checklist





Related to #
